clang/lib/Sema/SemaLookup.cpp

```diff
@@ -2390,7 +2390,8 @@ static bool LookupQualifiedNameInUsingDirectives(Sema &S, LookupResult &R,
 }
 
 bool Sema::LookupQualifiedName(LookupResult &R, DeclContext *LookupCtx,
-                               bool InUnqualifiedLookup) {
+                               bool InUnqualifiedLookup,
+                               IdentifierInfo const *IdentifierOverride) {
```

Comment:
What is the purpose of this change here   I don't see any changes in the lookup here where this is used.